ReactSkia/components/RSkComponentText.cpp

```diff
@@ -30,19 +31,19 @@ RSkComponentParagraph::RSkComponentParagraph(const ShadowView &shadowView)
     , expectedAttachmentCount(0)
     , currentAttachmentCount(0){}
 
-inline SkScalar yPosOffset(AttributedString attributedString, SkScalar paraHeight, Rect frame) {
+inline SkScalar yPosOffset(AttributedString attributedString, SkScalar paraHeight, Float frameHeight) {
     for(auto &fragment: attributedString.getFragments()) {
-        if((paraHeight < frame.size.height) && (!fragment.textAttributes.textAlignVertical.empty())) {
+        if((paraHeight < frameHeight) && (!fragment.textAttributes.textAlignVertical.empty())) {
             if(!strcmp(fragment.textAttributes.textAlignVertical.c_str(),"center"))
-                return ((frame.size.height/2) - (paraHeight/2));
+                return ((frameHeight/2) - (paraHeight/2));
             else if(!strcmp(fragment.textAttributes.textAlignVertical.c_str(),"bottom"))
-                return (frame.size.height - paraHeight);
+                return (frameHeight - paraHeight);
         }
     }
     return 0;
 }
 
-inline int getNumberOfLines(std::vector<LineMetrics>& metrics, int maxNumberOfLines, Float frameHeight) {
+inline int getMaxNumberOfLines(std::vector<LineMetrics>& metrics, int maxNumberOfLines, Float frameHeight) {
     int i=0;
     int minNumberOfLines = 0;
     for(; i< metrics.size(); i++) {
```

Comment:
better to calll getLines() here; to avoid duplication.

Reply:
Updated